wrapper/src/main/java/software/amazon/jdbc/util/SlidingExpirationCache.java

```diff
@@ -95,6 +95,18 @@ public V computeIfAbsent(
     return cacheItem.withExtendExpiration(itemExpirationNano).item;
   }
 
+  public V put(
+      final K key,
+      final V value,
+      final long itemExpirationNano) {
+    cleanUp();
+    final CacheItem cacheItem =  cache.put(key, new CacheItem(value, itemExpirationNano));
+    if (cacheItem == null) {
```

Comment:
That never happens.

Reply:
I was a bit surprised by this, but I've observed it happens on the first put (and im guessing if that k/v pair has been previously cleared).